examples/c/xmake.lua

```diff
@@ -0,0 +1,40 @@
+add_rules("mode.release", "mode.debug")
+add_rules("platform.linux.bpf")
+set_license("GPL-2.0")
+
+add_requires("linux-tools", {configs = {bpftool = true}})
+add_requires("libbpf")
+if is_plat("android") then
+    add_requires("ndk >=22.x", "argp-standalone")
+    set_toolchains("@ndk", {sdkver = "23"})
+else
+    add_requires("llvm >=10.x")
+    set_toolchains("@llvm")
+    add_requires("linux-headers")
+end
+
+add_includedirs("../../vmlinux")
+add_packages("linux-tools", "linux-headers", "libbpf")
```

Comment:
Does this download stuff from the internet?

Reply:
add_requires will do

Reply:
Downloading from the internet should be fine in the general case, but for libbpf-bootstrap we should use the vendored deps.